doc/findinfiles.rst

```diff
@@ -38,21 +39,25 @@ You can select from a number of options to enable searches as broad or refined a
 
 To enable case sensitivity, which only returns matches with identical capitalization to your search text, toggle the :guilabel:`Aa` button on.
 
-[GIF showing search with case sensitive activated]
+.. image:: images/find_in_files/find-in-files-case.gif
```

Comment:
I suggest you either make a cut during the search progress, or (better) speed it up by like 4x more. The GIF spends nearly 10 seconds sitting around waiting for the search to complete, most of that on an idle screen, by which point the user is likely to forget the exact original results and the intended purpose of the GIF (to show that there are fewer, different final results with case sensitivity enabled, not to show the search chugging along for 10 seconds). It also takes up almost 2 MB.

Reply:
Agreed.

Reply:
I'll cut a part from the search